### ISSUE TITLE
Outdated request dependency introduces security vulnerability

### ISSUE THREAD

**Issue by jbcpollak** [pusher/pusher-http-node#27]
I ran the `nsp` package against my project and it reported a vulnerability with this library:

```
(+) 1 vulnerabilities found
┌───────────────┬───────────────────────────────────────────────────────┐
│               │ Regular Expression Denial of Service                  │
├───────────────┼───────────────────────────────────────────────────────┤
│ Name          │ hawk                                                  │
├───────────────┼───────────────────────────────────────────────────────┤
│ Installed     │ 1.1.1                                                 │
├───────────────┼───────────────────────────────────────────────────────┤
│ Vulnerable    │ < 3.1.3  || >= 4.0.0 <4.1.1                           │
├───────────────┼───────────────────────────────────────────────────────┤
│ Patched       │ >=3.1.3 < 4.0.0 || >=4.1.1                            │
├───────────────┼───────────────────────────────────────────────────────┤
│ Path          │ mylib@0.0.1 > pusher@1.2.0 > request@2.45.0 > hawk@1.1.1  │
├───────────────┼───────────────────────────────────────────────────────┤
│ More Info     │ https://nodesecurity.io/advisories/77                 │
└───────────────┴───────────────────────────────────────────────────────┘
```

I noticed that if you upgrade to `request v2.69.0`, it depends on `hawk 3.1.3`, which is patched and does not have this vulnerability.

**Comment by jbcpollak** [comment]
I see there is already a pull request to solve this:

https://github.com/pusher/pusher-http-node/pull/26

**Comment by topliceanu** [comment]
Fixed in the above branch. I've published version 1.2.1 of the library.
```bash 
(+) No known vulnerabilities found
```
